frontend/src/components/charts/Chart.js

```diff
@@ -0,0 +1,29 @@
+import * as React from 'react'
+
+import styled from 'styled-components'
+import SimpleBarChart from './SimpleBarChart'
+import SimplePieChart from './SimplePieChart'
+
+export default function Chart({ variant, data }) {
+  return (
+    <StatistikContainer>
+      <ChooseChart variant={variant} data={data} />
+    </StatistikContainer>
+  )
+}
+function ChooseChart({ variant, data }) {
+  if (variant === 'SimpleBarChart') {
```

Comment:
maybe replace those Strings by constants to get autocomplete and reduce error-proneness